modules/microshift-deploying-operators.adoc

```diff
@@ -0,0 +1,15 @@
+// Module included in the following assemblies:
+//
+// * microshift_operators/microshift-deploying-operators.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-deploying-operators_{context}"]
+= Deploying operators using manifests
```

Comment:
You capitalized `Operators` in the heading of the assembly, but not in the heading of this module. I'm not sure which way is correct but wanted to point it out for consistency.

Reply:
Still needs a capital `O` here :slightly_smiling_face: